cassie/modules/empire.py

```diff
@@ -185,14 +177,15 @@ def cmd_empire_setup(self, args, jid, is_muc):
 			else:
 				report = '{0}: The specified URL is not valid.  Please enter it in the form of "https://127.0.0.1:1337/"'.format(report_user)
 			return report
-			
+
 		if results['show_config']:
 			if not report:
 				report = ''
 			report += 'Current Empire Config:\n'.format(report_user)
 			report += '\tUser: {0}\n'.format(user_storage.get('user', 'None'))
 			report += '\tPassword: {0}\n'.format(user_storage.get('pass', 'None'))
 			report += '\tURL: {0}\n'.format(user_storage.get('url', 'None'))
+			report += '\tVerify Cert: {0}\n.'format(user_storage.get('verify_cert', 'False'))
```

Comment:
This isn't necessary now because it'll never change. I'll remove it when I test it.